pages/docs/components/overlay/popover.mdx

```diff
@@ -240,18 +240,17 @@ prevent popover from returning focus to `PopoverTrigger`'s children.
 
 ```jsx
 function ControlledUsage() {
-  const [isOpen, setIsOpen] = React.useState(false)
-  const open = () => setIsOpen(!isOpen)
-  const close = () => setIsOpen(false)
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
   return (
     <>
-      <Button mr={5} onClick={open}>
+      <Button mr={5} onClick={onOpen}>
```

Comment:
Use `onToggle` here to reproduce the same behaviour as before :) Otherwise the popover will always stay open.

Reply:
Nice catch :wink: